packages/core/src/editor/BlockNoteEditor.ts

```diff
@@ -1038,4 +1038,16 @@ export class BlockNoteEditor<
       this._tiptapEditor.off("selectionUpdate", cb);
     };
   }
+
+  public openSelectionMenu(triggerCharacter: string) {
+    this.prosemirrorView.focus();
+    this.prosemirrorView.dispatch(
+      this.prosemirrorView.state.tr
+        .scrollIntoView()
+        .setMeta(this.suggestionMenus.plugin, {
```

Comment:
doesn't need to be `suggestionMenuPluginKey`?

Reply:
It's functionally the same thing, `setMeta` takes either a plugin key or the plugin itself